flake.nix

```diff
@@ -109,8 +110,8 @@
 
             # Shared environment variables
             env = {
-              CC = "clang";
-              CXX = "clang++";
+              CC = "${final.llzk-llvmPackages.clang}/bin/clang";
+              CXX = "${final.llzk-llvmPackages.clang}/bin/clang++";
```

Comment:
This change shouldn't be needed.

Reply:
IIUC, it's for the bootstrap version of clang/++ to build LLZK. The actual correct-version headers and such are included via what you added on lines 91 and 99.